### PR TITLE
App foundations

### PR DESCRIPTION
**Pull request by ljiang09**
Set file structure and added blank components for the app's pages. The Search page has upload image functionality

### CODE REVIEW

File: src/pages/AboutUs/index.jsx
Comment (aditivinod) [inline src/pages/AboutUs/index.jsx:1]: Can add placeholders for photos & short descriptions

File: src/pages/ImageDatabase/index.jsx
Comment (aditivinod) [inline src/pages/ImageDatabase/index.jsx:1]: Add clarification on whether these are past user-uploaded images or dataset images

**Review by aditivinod — CHANGES_REQUESTED** [review]
LGTM; good starting setup for the website (:

Add some documentation so that it is easier in the future to update/modify sections of the website.

image

File: src/pages/AboutUs/index.jsx
Comment (ljiang09) [reply src/pages/AboutUs/index.jsx:1]: Screen Shot 2023-09-25 at 12 53 39 AM

File: src/pages/ImageDatabase/index.jsx
Comment (ljiang09) [reply src/pages/ImageDatabase/index.jsx:1]: good idea

**Review by aditivinod — APPROVED** [review]

**Pull request merged**